tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/smt/Z3SolverContext.scala

```diff
@@ -374,8 +374,8 @@ class Z3SolverContext(val config: SolverConfig) extends SolverContext {
    *   the name of the new function (declared in SMT)
    */
   def declareCellFun(cellName: String, argType: CellT, resultType: CellT): Unit = {
-    val domSig = argType.signature
-    val resSig = resultType.signature
+    argType.signature
+    resultType.signature
```

Comment:
These can be removed, but that seems a bit suspicious too. I guess we should carefully consider this  code to ensure there's not a logic error?